packages/runtime/src/plugins/useSubgraphExecuteDebug.ts

```
@@ -11,41 +11,53 @@ export function useSubgraphExecuteDebug<
     onYogaInit({ yoga }) {
       fetchAPI = yoga.fetchAPI;
     },
-    onSubgraphExecute({ executionRequest, logger = opts.logger }) {
+    onSubgraphExecute({ executionRequest, logger = opts.logger, requestId }) {
       const subgraphExecuteId = fetchAPI.crypto.randomUUID();
-      logger = logger.child('subgraph-execute');
+      const subgraphExecuteHookLogger = logger.child({
+        subgraphExecuteId,
+        requestId: requestId!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)